app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,303 @@
+---
+title: MeshCircuitBreaker (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state, it should not be mixed
+with [CircuitBreaker](../circuit-breaker).
+{% endwarning %}
+
+This policy will look for errors in the live traffic being exchanged between our data plane proxies and it will mark a
+data proxy as an unhealthy if certain conditions are met and - by doing so - making sure that no additional traffic can
+reach an unhealthy data plane proxy until it is healthy again.
+
+Circuit breakers - unlike active [MeshHealthChecks](/docs/{{ page.version }}/policies/meshhealthcheck/) - do not send
+additional traffic to our data plane proxies but they rather inspect the existing service traffic. They are also
+commonly used to prevent cascading failures in our services.
+
+{% tip %}
+Like a real-world circuit breaker when the circuit is **closed** then traffic between a source and destination data
+plane proxy is allowed to freely flow through it, and when it is **open** then the traffic is interrupted.
+{% endtip %}
+
+The conditions that determine when a circuit breaker is closed or open are being configured on connection limits or
+outlier detection basis. For outlier detection to open circuit breaker you can configure what we call "detectors".
```

Comment:
should we go into more details about `outlier detection`?

Reply:
I would say it's explained deep enough as the next ~10 lines are about outlier detection